src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/view-data-brokers/page.tsx

```diff
@@ -2,25 +2,54 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+"use client";
+
 import Link from "next/link";
 import React from "react";
+import buttonStyles from "./../../../../../../../../components/server/button.module.scss";
+import styles from "./../dataBrokerProfiles.module.scss";
+import { useL10n } from "../../../../../../../../hooks/l10n";
 
 export default function ViewDataBrokers() {
+  const l10n = useL10n();
+
   return (
     <div>
-      ViewDataBrokers
-      <Link
-        href={
-          "/redesign/user/dashboard/fix/data-broker-profiles/automatic-remove"
-        }
-      >
-        Remove them for me
-      </Link>
-      <Link
-        href={"/redesign/user/dashboard/fix/data-broker-profiles/manual-remove"}
-      >
-        I’ll remove them manually
-      </Link>
+      <div className={styles.content}>
+        <h3>
+          {l10n.getString(
+            "fix-flow-data-broker-profiles-view-data-broker-profiles-headline",
+            { data_broker_sites_results_num: 78 }
+          )}
+        </h3>
+        <p>
+          {l10n.getString(
+            "fix-flow-data-broker-profiles-view-data-broker-profiles-content"
+          )}
+        </p>
+      </div>
+      <div className={styles.buttonsWrapper}>
+        <Link
+          className={`${buttonStyles.button} ${buttonStyles.primary}`}
+          href={
+            "/redesign/user/dashboard/fix/data-broker-profiles/automatic-remove"
+          }
```

Comment:
Nit: shouldn't need the `{}`

```suggestion
          href="/redesign/user/dashboard/fix/data-broker-profiles/automatic-remove"
```